tests/smoke-tests/test-relay-indices.ts

```diff
@@ -0,0 +1,226 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+import { Contract, ethers } from "ethers";
+import { getCompiled } from "../util/contracts";
+import { ALITH_ADDRESS, BALTATHAR_ADDRESS } from "../util/accounts";
+import { hexToU8a, u8aToHex } from "@polkadot/util";
+const debug = require("debug")("smoke:relay-indices");
+
+describeSmokeSuite(`Relay chain Module:Method indices should match our encoding`, (context) => {
+  let relayEncoder: Contract;
+  let xcmTransactorV1: Contract;
+  let xcmTransactorV2: Contract;
+  let rtVersion: number;
+
+  before(async function () {
+    if (process.env.SKIP_RELAY_TESTS) {
+      debug(`SKIP_RELAY_TESTS=true, skipping test.`);
+      this.skip();
+    }
+    rtVersion = context.polkadotApi.consts.system.version.specVersion.toNumber();
+
+    const RELAY_ENCODER_PRECOMPILE = "0x0000000000000000000000000000000000000805";
```

Comment:
We probably should put these in the util/constants.ts

Reply:
Convention amongst the other tests is to put precompile addresses at the top of file

Reply:
I'd say it depends which one. some precompile tests use the address from utils/constants.ts. Those tests that dont probably have not been adapted yet. But its not important, as you feel its better

Reply:
I don't mind the duplication of having this explicitly set everywhere we use it. In an ideal world we all remember the precompile addresses as a notable number, and we don't hide them in a constants file somewhere.